presto-main/src/main/java/com/facebook/presto/memory/ClusterMemoryManager.java

```diff
@@ -175,33 +179,22 @@ public synchronized void process(Iterable<QueryExecution> queries)
         }
         clusterMemoryUsageBytes.set(totalBytes);
 
-        if (killOnOutOfMemory) {
```

Comment:
Can we is  kill switch that disables this code entirelly?

Reply:
I'm happy to add an `instanceof NullKiller`